task-list-rails/app/views/tasks/show.html.erb

```diff
@@ -2,18 +2,68 @@
   <table>
     <thead>
         <tr>
-          <th colspan="2" id="title">Task details</th>
+          <td colspan="2" id="title">Task details</td>
+        </tr>
+        <tr>
+          <td colspan="2" id=edit_link><%= link_to "edit?", "/tasks/#{@task.id}/edit" %></td>
         </tr>
       </thead>
       <tbody>
-      <% @task_info_hash.each do |key, value| %>
-        <% if !value.nil? && value != "" %>
+        <tr>
+          <td class="attribute">
+            <strong>name:</strong>
+          </td>
+          <td>
+            <%= @task.name %>
+          </td>
+        </tr>
+        <tr>
+          <td class="attribute">
+            <strong>description:</strong>
+          </td>
+          <% if @task.description == "" || @task.description.nil? %>
```

Comment:
Unlike normal Ruby, Rails actually gives us a useful method for combining these two checks into one thing: `present?`. You can call this method on `nil` or any other value.